applications/nrf_desktop/src/util/CMakeLists.txt

```diff
@@ -6,6 +6,17 @@
 target_sources_ifdef(CONFIG_DESKTOP_HWID
 		     app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/hwid.c)
 
+if(CONFIG_DESKTOP_HWINFO_BLE_ADDRESS_FICR)
```

Comment:
You can slightly update commit message here (you no longer need to disable the `nRF hwinfo` driver).